.github/workflows/cypress_workflow_with_s3.yml

```diff
@@ -71,18 +64,11 @@ jobs:
       - name: Build plugins
         run: node scripts/build_opensearch_dashboards_platform_plugins --no-examples --workers 12
 
-      - name: Checkout FT repo
-        uses: actions/checkout@v2
-        with:
-          path: ${{ env.FTR_PATH }}
-          repository: ${{ env.TEST_REPO }}
-          ref: '${{ env.TEST_BRANCH }}'
-
       - name: Setup spec files
         if: ${{ inputs.specs == '' }}
         shell: bash
         run: |
-          DASHBOARDS_SPEC="$(yarn --silent osd:ciGroup10)"
+          DASHBOARDS_SPEC="cypress/integration/core_opensearch_dashboards/opensearch_dashboards/apps/query_enhancements/04/s3_dataset.spec.js"
```

Comment:
should this just be in a separate folder?

Reply:
I think that would be ideal, since then it wouldn't get run with the other tests during PRs and speed up some of the CI downtime. I don't have too much context on how the current directories are set up for our tests, do you know where exactly would be a good place to add a new directory just for this test? I could then update the `package.json` to have a command for this.

Reply:
Yeah would prefer another ciGroup for s3 tests.

Reply:
or `"osd:s3Tests": "echo \"cypress/integration/core_opensearch_dashboards/opensearch_dashboards/apps/s3_tests/*.spec.js\""`